pkg/cmap/cmap_test.go

```
@@ -0,0 +1,213 @@
+			m.Set(i, i)
+		}
+
+		var wg sync.WaitGroup
+		wg.Add(3) // For Keys, Values, and modifier goroutines
+
+		// Start a goroutine to continuously modify the map.
+		go func() {
+			defer wg.Done()
+			for i := 0; i < numItems; i++ {
+				m.Set(randomIntn(numItems), i)
+				m.Delete(randomIntn(numItems), func(value int, exists bool) bool {
```

golangci-lint

[warning] unused-parameter: parameter 'value' seems to be unused, consider removing or renaming it as _

(revive)